README.md

```diff
@@ -27,44 +27,37 @@ Install Anaconda:  https://www.anaconda.com/download/ Python 3.6 version
 
 Anaconda Prompt may be used for the following
 
-Create a virtual environment (preferably containing a username, example below is for sctest as the name of the environment)
+Create a virtual environment using the environment.yml (will create an environment called silcam)
 
 ```
-    conda create -n <name of the environment> python=3.5
```

Comment:
Is the README consistent on which python version is required now?

Reply:
yes